modules/op-about-tasks.adoc

```diff
@@ -40,7 +40,3 @@ spec: <4>
 <2> Specifies the type of Kubernetes object. In this example, `Task`.
 <3> Unique name of this Task.
 <4> Lists the parameters and Steps in the Task and the workspace used by the Task.
-
-This Task starts the pod and runs a container inside that pod using the `maven:3.6.0-jdk-8-slim` image to run the specified commands. It receives an input directory called `workspace-git` that contains the source code of the application.
-
-The Task only declares the placeholder for the Git repository, it does not specify which Git repository to use. This allows Tasks to be reusable for multiple Pipelines and purposes.
```

Comment:
I am not sure we need to remove this information, AFAIK this is still relevant. @piyush-garg can you confirm please?

Reply:
@piyush-garg @Preeticp Please review this information in light of the information @hupiper provides in the original issue #30364. 

I might be wrong, but the two phrases we are removing seem to refer to elements of the example in https://openshift.github.io/pipelines-docs/docs/0.10.5/proc_defining-and-creating-pipeline-tasks.html:
- `image: maven:3.6.0-jdk-8-slim`
- `type: git`

These elements are not present in the example along side the phrases we are removing https://docs.openshift.com/container-platform/4.7/cicd/pipelines/understanding-openshift-pipelines.html#about-tasks_understanding-openshift-pipelines.

Reply:
Yes @Preeticp @rolfedh Seems like the docs.openshift.com have wrong info there, we may need to tweak this based on the example there, but let's not remove it

Reply:
The parameter is correct now.I'm not sure we're showing the whole pipeline in context,, but it will work for now.

There is at least one more grammatical problem I caught when looking over the example. I'll submit that as an issue.